client/blocks/comment-detail/comment-detail-author.jsx

```diff
@@ -32,6 +33,25 @@ export class CommentDetailAuthor extends Component {
 		this.setState( { isExpanded: ! this.state.isExpanded } );
 	};
 
+	commentStatusLabel = () => {
+		const { commentStatus, translate } = this.props;
+		if ( 'approved' === commentStatus ) {
```

Comment:
Total aide, it's probably time for us to add these comment statuses as an enum in a comment util. This can wait for another janitorial PR.